### PR TITLE
Implement the ModelManager.ModelInfo client API call.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4819/
Test PASSed.

File: jujugui/static/gui/src/app/store/env/go.js [hunk 1]
```diff
@@ -697,6 +697,71 @@ YUI.add('juju-env-go', function(Y) {
...
+          console.log('data returned by model info API call:', data);
+          return;
+        }
+        var err = data.Error && data.Error.Message;
+        if (err) {
+          userCallback({err: err});
+          return;
+        }
+        var results = data.Response.results;
+        if (results.length != 1) {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/store/env/go.js]: `!==`?

**Comment by huwshimi** [comment]
Looks good @frankban :+1: with one trivial comment.

File: jujugui/static/gui/src/app/store/env/go.js (hunk 1 above)
Comment (frankban) [inline jujugui/static/gui/src/app/store/env/go.js]: Done.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4831/
Test PASSed.

**Comment by frankban** [comment]
Thanks for the review Huw!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**